### ISSUE TITLE
TypeScript Related Error Message

### ISSUE THREAD

**Issue by lewismcruz** [microsoft/vscode#98926]
I have simply been coding JavaScript.js practice files, but have not been running them in any browser or application.  They have simply been static files to help me familiarize myself with JavaScript syntax.  

I was not doing anything except typing out JavaScript code when VS Code threw up an error message related to some sort of TypeScript error or functionality that had failed 5 times (to run in the background).  

I have Chrome open with about 40+ plus tabs and Opera with about another 30 tabs but otherwise was just typing my practice code files and saving them locally when this error message popped up, so I am not really sure what I did to cause VS Code to throw this message.

Running Windows 7 Ultimate 64-bit edition.

This is the first time I have experienced this issue since using VS Code for about 5 weeks now.  I have not done anything different than open VS Code and create new .js and .html files and have generally enjoyed coding using VS Code.

**Comment by egamma** [comment]
/needsMoreInfo can you please share the workspace where you have experienced this problem?

Is there any more info in the typescript log (Output panel at the bottom, select TypeScript from the dropdown on the right).

**Comment by lewismcruz** [comment]
Hi Erich,

Here are more details regarding my installation of VS Code and background
information.  I hope it is useful to you.

Workspace I have been working in is: Codingbootcampwrkspace

Attached is a screenshot of extensions I have installed and have enabled.
I wish I would have grabbed the exact working of the error message and
tried to decipher the exact wording of it before closing it.

Again -  I was not doing anything but editing or creating new JavaScript
files to practice coding, so I was puzzled what I might have done to
generate the errors.  They seemed to occur spontaneously.  And since that
error occurred I have restarted my VS Code session about two times and have
been working with my files without further issues.

I have used Visual Studio in the past, but really enjoy coding with VS
Code.  It seems easier to use and not overwhelming to use.

Thanks for contacting me regarding this issue. I wish I could be of more
help.  My VS Code is working fine otherwise, but if the issue occurs again
I will attempt to document it better then.

I believe I have been running the Electron vue theme when the issue first
occurred:

[image: image.png]

[image: image.png]

[image: image.png]
